scripts/zones/Garlaige_Citadel/mobs/Serket.lua

```diff
@@ -22,7 +22,11 @@ end
 
 entity.onMobDespawn = function(mob)
     UpdateNMSpawnPoint(mob:getID())
-    mob:setRespawnTime(math.random(3600, 43200)) -- 1 to 12 hours
+    if xi.settings.main.ENABLE_WOTG == 1 then
```

Comment:
I think this happened after ASB cut-off date

Reply:
This is correct, I'm still seeing 21-24 hr on May 2010